src/Task.js

```diff
@@ -0,0 +1,7 @@
+export default class Task {
```

Comment:
- [ ] It would be better if we have a more descriptive README file, kindly check this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) to have access to the template and see how best we can improve on ours